src/mlops_codex/external_monitoring.py

```diff
@@ -129,19 +140,30 @@ def upload_file(
         shap_reference: Optional[str] = None,
         python_version: Optional[str] = "3.10",
     ):
-        """Validate inputs before send files
-
-        Args:
-            model_file (Optional[str], optional): Path to your model.pkl file. Defaults to None.
-            requirements_file (Optional[str]): Path to your requirements.txt file. Defaults to None.
-            preprocess_file (Optional[str]): Path to your preprocessing file. Defaults to None.
-            preprocess_reference (Optional[str]): Preprocessing function entrypoint. Defaults to None.
-            shap_reference (Optional[str]): Shap function entrypoint. Defaults to None.
-            python_version (Optional[str], optional): Python version. Can be "3.8", "3.9" or "3.10". Defaults to "3.10".
-
-        Raises:
-            InputError
-            InputError
+        """
+        Validate inputs before sending files.
+
+        Parameters
+        ----------
+        model_file : Optional[str], optional
+            Path to your `model.pkl` file. Defaults to None.
+        requirements_file : Optional[str], optional
+            Path to your `requirements.txt` file. Defaults to None.
+        preprocess_file : Optional[str], optional
+            Path to your preprocessing file. Defaults to None.
+        preprocess_reference : Optional[str], optional
+            Preprocessing function entrypoint. Defaults to None.
+        shap_reference : Optional[str], optional
+            Shap function entrypoint. Defaults to None.
+        python_version : Optional[str], optional
+            Python version. Can be "3.8", "3.9", or "3.10". Defaults to "3.10".
```

Comment:
I noticed that there's some inconsistency for the default python version. This is something we can improve for the codex. What you think @johnrc90 @henrique-lh?

Reply:
Agree, I updated this, now the default version is Python 3.10 for every function. Maybe, we shouldn't even have a default value for this in my opinion

Reply:
https://github.com/datarisk-io/mlops_codex/pull/68/commits/13337399b3e7eb948bd2e048488934f7eadfd110